### PR TITLE
Update with new Pages _headers feature

### PR DESCRIPTION
**Pull request by michaelkennedy**

### CODE REVIEW

File: products/pages/src/content/how-to/add-custom-http-headers.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ pcx-content-type: how-to
 
 # Add custom HTTP headers
 
-Cloudflare provides HTTP header customization for Pages projects through Cloudflare Workers [serverless functions](https://www.cloudflare.com/learning/serverless/what-is-serverless/). You may want to add, remove, or alter HTTP headers for security or performance reasons. 
+Cloudflare provides HTTP header customization for Pages projects by ading a `_headers` file to your project. Check out the [documentation](https://developers.cloudflare.com/pages/platform/headers). You may want to add, remove, or alter HTTP headers for security or performance reasons.
```
Comment (WalshyDev) [inline products/pages/src/content/how-to/add-custom-http-headers.md]: I would recommend making this a note so it stands out more. Also fixed a typo

```suggestion
<Aside type="note">

Cloudflare provides HTTP header customization for Pages projects by adding a `_headers` file to your project. Check out the [documentation](https://developers.cloudflare.com/pages/platform/headers). You may want to add, remove, or alter HTTP headers for security or performance reasons.

</Aside>
```

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: products/pages/src/content/how-to/add-custom-http-headers.md [hunk 2]
```diff
@@ -4,7 +4,13 @@ pcx-content-type: how-to
 
 # Add custom HTTP headers
 
-Cloudflare provides HTTP header customization for Pages projects through Cloudflare Workers [serverless functions](https://www.cloudflare.com/learning/serverless/what-is-serverless/). You may want to add, remove, or alter HTTP headers for security or performance reasons. 
+<Aside type="note">
+
+Cloudflare provides HTTP header customization for Pages projects by adding a `_headers` file to your project. Check out the [documentation](https://developers.cloudflare.com/pages/platform/headers). You may want to add, remove, or alter HTTP headers for security or performance reasons.
```
Comment (deadlypants1973) [inline products/pages/src/content/how-to/add-custom-http-headers.md]:
```suggestion
Cloudflare provides HTTP header customization for Pages projects by adding a `_headers` file to your project. Refer to the [documentation](https://developers.cloudflare.com/pages/platform/headers) for more information.
```

**Review by deadlypants1973 — APPROVED** [review]
PCX review

File: products/pages/src/content/how-to/add-custom-http-headers.md [hunk 3]
```diff
@@ -4,7 +4,13 @@ pcx-content-type: how-to
...
 # Add custom HTTP headers
 
-Cloudflare provides HTTP header customization for Pages projects through Cloudflare Workers [serverless functions](https://www.cloudflare.com/learning/serverless/what-is-serverless/). You may want to add, remove, or alter HTTP headers for security or performance reasons. 
+<Aside type="note">
+
+Cloudflare provides HTTP header customization for Pages projects by adding a `_headers` file to your project. Refer to the [documentation](https://developers.cloudflare.com/pages/platform/headers) for more information.
+
+</Aside>
+
+More advanced customisation of HTTP headers is available through Cloudflare Workers [serverless functions](https://www.cloudflare.com/learning/serverless/what-is-serverless/).
```
Comment (deadlypants1973) [inline products/pages/src/content/how-to/add-custom-http-headers.md]:
```suggestion
More advanced customization of HTTP headers is available through Cloudflare Workers [serverless functions](https://www.cloudflare.com/learning/serverless/what-is-serverless/). 
```

**Review by kristianfreeman — APPROVED** [review]

**Review by jkup — APPROVED** [review]
Thanks for this!

**Pull request merged**